src/events.jl

```diff
@@ -44,35 +44,52 @@ end
 ```
 """
 function event_loop(callback::Function, plotobj::DisplazWindow, event_list...)
+  
+    command = hook_command(plotobj.name, event_list...)
+
+    open(command, "r") do stdout
+        handle_events(callback, stdout)
```

Comment:
In Julia 1.0, the global standard stream names were renamed form `STDOUT` to `stdout`, etc. So let's use some other name for the local variable (here and in the functions below) to avoid confusing it with `Base.stdout`

```suggestion
    open(command, "r") do event_stream
        handle_events(callback, event_stream)
```